app/controllers/term_controller.rb

```diff
@@ -3,8 +3,15 @@
 class TermController < ApplicationController
   include Pagy::Backend
 
-  # This provides the list of terms that are awaiting confirmation.
+  # This provides the list of terms that are awaiting confirmation. By default this shows only terms which have been
+  # categorized automatically. Adding `type=all` to the querystring will show _all_ terms which the user has not yet
+  # confirmed.
```

Comment:
We don't need to change for this PR as the default is what we expect people to use for the foreseeable future, but it might be better longterm to make this a sticky option by setting it as a session variable or cookie. I don't even think we need a ticket for that as we may never come back to it, but just wanted to share that the ergonomics of this is a bit awkward if someone is trying to "categorize" terms rather than validate categorizations.

Reply:
Yeah, having the choice be sticky seems like something that could be useful in the future, or even a workflow that just picks terms according to some logic, so that the user never goes back to a list-of-terms page.

Regardless, I agree that this is a concern for the future should we ever really start using this workflow.